### PR TITLE
DOP-3304, DOP-3335, DOP-3359: update manifest job handler to use snooty manifest

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +34,9 @@ RUN /venv/bin/pip install -r https://raw.githubusercontent.com/mongodb/docs-tool
 RUN apt-get update && apt-get install -y python3 python3-dev python3-pip
 RUN apt-get -y install vim
 RUN apt-get -y install git pkg-config libxml2-dev
-RUN python3 -m pip install -r https://raw.githubusercontent.com/mongodb/mut/master/requirements.txt
+RUN python3 -m pip install https://github.com/mongodb/mut/releases/download/v0.10.1/mut-0.10.1-py3-none-any.whl
```
Comment (i80and) [inline Dockerfile:37]: Tasty.

There's the theoretical issue that we're no longer pinning dependencies, but I don't think we need to overthink this.

**Review by i80and — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (schmalliso) [reply Dockerfile:37]: Are they not semi-pinned in [pyproject.toml](https://github.com/mongodb/mut/blob/master/pyproject.toml) ?

Comment (i80and) [reply Dockerfile:37]: that's a dependency specification, not pinning per se. The idea of pinning is to express "developers have tested *this exact* dependency version and it is known to be good", preventing a regression in a dependency from hosing things.

**Comment by schmalliso** [comment]
@i80and Rerequesting since I've touched a bunch of other code since your original LGTM as I found _opportunities_ for enhancement and also bugs.

File: api/controllers/v1/slack.ts [hunk 2]
```diff
@@ -140,9 +140,11 @@ export const getDeployableJobs = async (values, entitlement, branchRepository: B
       urlSlug,
       false,
       false,
-      '-g'
+      false
```
Comment (i80and) [inline api/controllers/v1/slack.ts:143]: Do I want to know? 😅

File: api/controllers/v1/slack.ts [hunk 3]
```diff
@@ -140,9 +140,11 @@ export const getDeployableJobs = async (values, entitlement, branchRepository: B
...
     );
 
+    newPayload.stable = isStableBranch ? true : false;
```
Comment (i80and) [inline api/controllers/v1/slack.ts:146]: I think the `!!isStableBranch` incantation is idiomatic for this. or `new Boolean(isStableBranch)`.

File: api/controllers/v1/slack.ts (hunk 2 above)
Comment (schmalliso) [reply api/controllers/v1/slack.ts:143]: You do not.

File: api/controllers/v1/slack.ts (hunk 3 above)
Comment (schmalliso) [reply api/controllers/v1/slack.ts:146]: ooh nice. done.

**Comment by schmalliso** [comment]
Checks passed on the non-forked Draft. https://github.com/mongodb/docs-worker-pool/pull/711 Awaiting Cassidy feedback before I click the 'Merge without waiting for requirements' button like a champ.

**Review by casthewiz — APPROVED** [review]
LGTM - I'm increasingly questioning the utility of the `createPayload` function in the slack controller, but that's a revision for another PR imo.

**Pull request merged**